file_entity.pages.inc

```diff
@@ -1,87 +0,0 @@
-<?php
-
-/**
- * @file
- * Supports file operations including View, Edit, and Delete.
- */
-
-function file_entity_upload_archive_form($form, &$form_state) {
```

Comment:
WRT hook_menu removal see https://trello.com/c/3nOg4Cpn/37-fe-task-port-code-that-is-left-in-hook-menu-from-d7. 

Summary: Create issue in token issue queue and remove hook_menu from file_entity.